src/EnergyPlus/AirflowNetwork/include/AirflowNetwork/Solver.hpp

```diff
@@ -72,6 +72,27 @@ struct EnergyPlusData;
 
 namespace AirflowNetwork {
 
+    // Vent Control  DistSys Control  Flag    Description
+    //  NONE           NONE           0      No AirflowNetwork and SIMPLE
+    //  SIMPLE         NONE           1      Simple calculations only
+    //  MULTIZONE      NONE           2      Perform multizone calculations only
+    //  NONE           DISTSYS        3      Perform distribution system during system on time only
+    //  SIMPLE         DISTSYS        4      Perform distribution system during system on time and simple calculations during off time
+    //  MULTIZONE      DISTSYS        5      Perform distribution system during system on time and multizone calculations during off time
+    //
+    // int constexpr AirflowNetworkControlSimple(1);    // Simple calculations only
+    // int constexpr AirflowNetworkControlMultizone(2); // Perform multizone calculations only
+    // int constexpr AirflowNetworkControlSimpleADS(4); // Perform distribution system during system on time and simple calculations during off time
+    // int constexpr AirflowNetworkControlMultiADS(5);  // Perform distribution system during system on time and multizone calculations during off time
+
+    enum class ControlType
+    {
+        NoMultizoneOrDistribution = 0,                   // Simple calculations only, AirflowNetworkControlSimple(1)
+        MultizoneWithoutDistribution,                    // Perform multizone calculations only, AirflowNetworkControlMultizone(2)
+        MultizoneWithDistributionOnlyDuringFanOperation, // Perform distribution system during system on time and simple calculations during off time, AirflowNetworkControlSimpleADS(4)
+        MultizoneWithDistribution                        // Perform distribution system during system on time and multizone calculations during off time, AirflowNetworkControlMultiADS(5)
+    };
+
```

Comment:
Here's the replacement enumeration. I've kept some of the original information around for reference, we can delete some of this later.